src/goose/cli/config.py

```diff
@@ -87,11 +87,6 @@ def default_model_configuration() -> Tuple[str, str, str]:
             break
         except Exception:
             pass
-    else:
```

Comment:
we don't need this message.  The error message (env var is not set) will be throw when the provider is loaded while creating the session.

Reply:
so in this case, the provider value that is returned below will be whatever is last in the list of providers, and then be used to set the default profile. It will later error like you say. 

Should we do 
```
else:
    provider = RECOMMENDED_DEFAULT
```
so that we don't set it to something niche?